main.js

```diff
@@ -0,0 +1,64 @@
+const container = document.querySelector('.container');
+const form = document.getElementById('form');
+
+let myBooks = [];
+if (localStorage.getItem('Data-base') !== null) {
+  myBooks = JSON.parse(localStorage.getItem('Data-base'));
+}
+
+function Book(title, author, id) {
+  this.title = title;
+  this.author = author;
+  this.id = id;
+}
+
+function addBookToLibrary(book) {
+  const bookDiv = document.createElement('div');
+  bookDiv.classList.add('bookInfos');
+  const h2 = document.createElement('h2');
+  const h3 = document.createElement('h3');
+  const removeBtn = document.createElement('button');
+  removeBtn.classList.add('remove');
+  h2.textContent = book.title;
+  h2.classList.add('book-title');
+  h3.textContent = book.author;
+  h3.classList.add('book-author');
+  removeBtn.textContent = 'Remove';
+  removeBtn.classList.add('btn-1');
+  removeBtn.setAttribute('id', book.id);
+  bookDiv.appendChild(h2);
+  bookDiv.appendChild(h3);
+  bookDiv.appendChild(removeBtn);
+  container.appendChild(bookDiv);
+}
+function displayBooks() {
+  myBooks.forEach((element) => {
+    addBookToLibrary(element);
+  });
+}
+
+function removeBook(book) {
+  const index = myBooks.findIndex((books) => books.id === book);
+  myBooks.splice(index, 1);
```

Comment:
- [ ] Favor the use of array methods that don't mutate the array, e.g`. concat, filter, slice` instead of push, pop, splice, and so on. This is common as a good practice and helps for debugging.